packages/path_provider/path_provider_linux/test/get_application_id_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:path_provider_linux/src/get_application_id_real.dart';
+
+class FakeGioUtils implements GioUtils {
+  int? application = 0;
```

Comment:
```suggestion
  int application = 0;
```

Reply:
Fixed the other way, by removing the default, which is what I meant to do. Originally I made them non-nullable with default values but then it's possible to have tests that are expecting things based on setup that was implicit elsewhere, which makes tests harder to understand. I meant to change them to nullable without defaults to prevent that possibility, but forgot to remove the defaults.